### PR TITLE
Updated virtualization support info for 15 SP6

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Are there any relevant issues/feature requests?

* jsc#PED-5612

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2

- SLE 12
  - [ ] SLE 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Review by aginies — APPROVED** [review]
sounds ok. Thanks for the update

File: xml/virt_support.xml [hunk 1]
```diff
@@ -821,9 +1028,18 @@
...
+        <title>Confidential Computing</title>
+        <para>
+          &slsa; 15 SP6 and newer include kernel patches and tooling in to add
+          AMD and Intel Confidential computing technology in the product. As
+          this technology is not yet fully ready for production environment, it
+          is provided as technology preview.
+        </para>
+      </important>
       <table>
         <title>Supported live-migration guests</title>
```
Comment (dariavladykina) [inline xml/virt_support.xml:1041]:
```suggestion
        <title>Supported live migration guests</title>
```

File: xml/virt_support.xml [hunk 2]
```diff
@@ -756,7 +962,8 @@
               </entry>
             </row>
             <row>
-              <entry>(<phrase role="color:suse-jungle-green">✓</phrase>)</entry>
+              <entry><phrase role="color:suse-jungle-green">✓</phrase><superscript>1</superscript>
+              </entry>
               <entry>
                 <para>
                   Supported for &kvm; hypervisor only.
```
Comment (dariavladykina) [inline xml/virt_support.xml:969]:
```suggestion
                  Supported for &kvm; hypervisor only
```

File: xml/virt_support.xml [hunk 3]
```diff
@@ -426,9 +548,8 @@
           All guest operating systems are supported both fully virtualized and
           paravirtualized. The exception is Windows systems, which are only
           supported fully virtualized (but they can use PV drivers:
```
Comment (dariavladykina) [inline xml/virt_support.xml:550]:
```suggestion
          paravirtualized. The exception is Windows systems, which 
          require full virtualization for support (but they can use PV drivers:
```

File: xml/virt_support.xml [hunk 4]
```diff
@@ -312,6 +310,140 @@
...
+        <para>
+          Windows Server 2016, 2019, 2022
+        </para>
+      </listitem>
+    </itemizedlist>
+
+    <para>
+      You can also search in the
+      <link xlink:href="https://www.suse.com/yessearch/Search.jsp">SUSE YES
+      certification database</link>
```
Comment (dariavladykina) [inline xml/virt_support.xml:420]:
```suggestion
      certification database</link>.
```

File: xml/virt_support.xml [hunk 5]
```diff
@@ -312,6 +310,140 @@
...
+    </para>
+
+    <itemizedlist>
+      <title>The level of support is as follows</title>
+      <listitem>
+        <para>
+          Support for SUSE host operating systems is full L3 (both for the guest
+          and host) according to the respective
```
Comment (dariavladykina) [inline xml/virt_support.xml:428]:
```suggestion
          and host), according to the respective
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

**Pull request merged**